Content.Shared/_RMC14/Sentry/SentryComponent.cs

```diff
@@ -48,7 +48,7 @@ public sealed partial class SentryComponent : Component
     public string ContainerSlotId = "gun_magazine";
 
     [DataField, AutoNetworkedField]
-    public EntProtoId[]? Upgrades = ["RMCSentryMini"];
+    public EntProtoId[]? Upgrades = ["RMCSentryMini", "RMCSentrySniper", "RMCSentryOmni"];
```

Comment:
Small nitpick so the order more closely resembles 13's
```suggestion
    public EntProtoId[]? Upgrades = ["RMCSentrySniper", "RMCSentryMini", "RMCSentryOmni"];
```